Machine.Specifications.ReSharper.Provider.10/MspecTestMetadataExplorer.cs

```diff
@@ -30,25 +30,26 @@ public void ExploreAssembly(IMetadataAssembly assembly, CancellationToken token)
 
                 foreach (var type in types)
                 {
-                    InterruptableActivityCookie.CheckAndThrow();
-                    token.ThrowIfCancellationRequested();
+                    if (token.IsCancellationRequested)
+                        break;
```

Comment:
Not sure if this is technically correct (xunit throws instead of breaks), but it's a real pain to debug with this code constantly throwing. And it doesn't seem to impact anything.